src/util/format.ts

```diff
@@ -66,6 +66,13 @@ export function encodeHTML(source: string): string {
         });
 }
 
+export function concatTooltipHtml(html: string, value: unknown, dontEncodeHtml = false): string {
```

Comment:
In my opinion, do not use language feature of "default parameters", especially for a "public" util method.

echarts code trade those three things the same:
`null`, `undefined` and "no that parameter / no that property".
I think it's not a good practice to distinguish them and not necessary.
But the language feature of "default parameters" only works for undefined and no parameter, but not for null.

Reply:
Thanks for this,  I actually don't know "default parameters" doesn't work for `null` :sweat_smile: